README.md

```diff
@@ -38,17 +38,18 @@ The following sample creates an authentication provider that [gets access withou
 ```php
 
 use Microsoft\Kiota\Authentication\Oauth\ClientCredentialContext;
-use Microsoft\Graph\Core\Authentication\GraphPhpLeagueAuthenticationProvider;
+use Microsoft\Kiota\Authentication\PhpLeagueAuthenticationProvider;
 
 $tokenRequestContext = new ClientCredentialContext(
     'tenantId',
     'clientId',
     'clientSecret'
 );
 // uses https://graph.microsoft.com/.default scopes
-$authProvider = new GraphPhpLeagueAuthenticationProvider($tokenRequestContext);
+$authProvider = new PhpLeagueAuthenticationProvider($tokenRequestContext);
```

Comment:
Hesitant on undoing this and the RC version since our next release will need them